ush/soca/obs_list_test.yaml

```diff
@@ -0,0 +1,58 @@
+observers:
+- !INC ${OBS_YAML_DIR}/adt_3a_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_3b_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_6a_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_all.yaml
+- !INC ${OBS_YAML_DIR}/adt_c2_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_coperl4.yaml
+- !INC ${OBS_YAML_DIR}/adt_j2_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_j2.yaml
+- !INC ${OBS_YAML_DIR}/adt_j3_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/adt_j3.yaml
+- !INC ${OBS_YAML_DIR}/adt_sa_egm2008.yaml
+- !INC ${OBS_YAML_DIR}/bufr_sfcshipsu.yaml
+- !INC ${OBS_YAML_DIR}/bufr_sfcships.yaml
+- !INC ${OBS_YAML_DIR}/icec_amsr2_north.yaml
+- !INC ${OBS_YAML_DIR}/icec_amsr2_south.yaml
+- !INC ${OBS_YAML_DIR}/icec_emc.yaml
+- !INC ${OBS_YAML_DIR}/icec_nsidc_nh.yaml
+- !INC ${OBS_YAML_DIR}/icec_nsidc_sh.yaml
+- !INC ${OBS_YAML_DIR}/icec_ssmis_f17_north.yaml
+- !INC ${OBS_YAML_DIR}/icec_ssmis_f17_south.yaml
+- !INC ${OBS_YAML_DIR}/icec_ssmis_f18_north.yaml
+- !INC ${OBS_YAML_DIR}/icec_ssmis_f18_south.yaml
+- !INC ${OBS_YAML_DIR}/icefb_gdr.yaml
+- !INC ${OBS_YAML_DIR}/oc_aqua.yaml
```

Comment:
we don't need ocean color yamls.
Please remove `oc_aqua.yaml, oc_dineof3.yaml and oc_snpp.yaml`

Reply:
oc_noaa20 too?

Reply:
@AndrewEichmann-NOAA, yes please

Reply:
done